kaldb/src/main/java/com/slack/kaldb/logstore/LuceneIndexStoreImpl.java

```diff
@@ -59,12 +60,13 @@
   private final Counter commitsCounter;
   private final Counter refreshesCounter;
 
-  public static LuceneIndexStoreImpl makeLogStore(File dataDirectory, MeterRegistry metricsRegistry)
+  public static LuceneIndexStoreImpl makeLogStore(
+      File dataDirectory, KaldbConfigs.IndexerConfig indexerConfig, MeterRegistry metricsRegistry)
```

Comment:
This should just take the commit duration and refresh duration if those are the only arguments we need.

Reply:
Passing in most general type here in case we add more params in future. Ideally, we should refactor these params into it's own Lucene config.

Reply:
done.

Reply:
It looks like this wasn't entirely updated, we didn't update the method to take the newly created `LuceneConfig`?